pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -131,6 +131,19 @@ def test_get_data_multiple_symbols(self):
         sl = ['AAPL', 'AMZN', 'GOOG']
         web.get_data_yahoo(sl, '2012')
 
+    @pytest.mark.parametrize('adj_pr', [True, False])
+    def test_get_data_null_as_missing_data(self, adj_pr):
+        # workaround as skip_on_exception decorator changes signature
+        # should it be changed to a signature-preserving decorator?
```

Comment:
> I have looked at the decorator package not sure how to use it in our case,

IMO, using another package seems a bit overkill.  I imagine that the fix for this issue shouldn't need to require another package just for this one decorator.